src/v/security/authorizer.h

```diff
@@ -131,6 +124,42 @@ class authorizer final {
     }
 
 private:
+    /*
+     * Compute whether the implied operations are allowed based on the specified
```

Comment:
I think maybe the comment is backwards, or I'm not parsing it correctly?

Isn't it:

Determine whether the given operation is allowed based on the implied allowed operations in acl_matches.

?

Reply:
Yes you are right - fixing.